app/templates/signup.hbs

```diff
@@ -1,45 +1,74 @@
 {{page-title "Sign Up"}}
 
-<form class="form">
-  <h2>{{this.title}}</h2>
-  {{#each this.fields as |field|}}
-  {{#let this.formData as | myFormData |}}
-    <FormInput
-      @id={{field.id}}
-      @for={{field.for}}
-      @label={{field.label}}
-      @type={{field.type}}
-      @value={{get myFormData field.id}}
-      @placeholder={{field.placeholder}}
-      @errorMessage={{field.errorMessage}}
-      @required={{field.required}}
-      @showError={{field.showError}}
-      @helpMsg={{field.helpMsg}}
-      @onChange={{this.handleFieldChange}}
-      @validator={{field.validator}}
-    />
-    {{/let}}
-  {{/each}}
-  {{#if this.isSubmitDisabled }}
-    <button
-      class='submitButton disabledButton'
-      type="submit"
-      disabled={{true}}
-    >
-      Submit
-    </button>
+{{#if this.dev}}
+  {{#if this.state}}
```

Comment:
To see the new flow, I'll have to pass `?dev=true&state=get-started` right?

What if I pass just `?dev=true`? Neither the new nor the new flow will trigger, right? Because we don't have a default value for `state`

Is my understanding correct or I'm missing something?

Reply:
Added a default value - `get-started`